src/app/components/content/partner/partner.component.scss

```diff
@@ -0,0 +1,70 @@
+//#buttonlogin{
+//  position: absolute;
+//  right: 50px;
+//  width: 100px;
+//}
+
+.form-login{
+  padding: 10px 30px;
+  max-width: 500px;
```

Comment:
I'm personally not a huge fan of using pixels as measurement unit, using something like `em` or `rem` (like bootstrap) ensures that our application is a little more responsive when changing the general font-size.

Reply:
I can change it to em or rem, but with padding and margin I always use pixels